plugins/source/alicloud/client/spec.go

```diff
@@ -0,0 +1,8 @@
+package client
+
+type Spec struct {
+	RegionID    string `json:"region_id,omitempty"`
+	AccessKey   string `json:"access_key,omitempty"`
+	SecretKey   string `json:"secret_key,omitempty"`
+	BillHistory int    `json:"bill_history,omitempty"`
```

Comment:
What does this option do? Looks like it either loads the billing history or only the latest bill? Should it be a boolean in that case? I think we'll probably soon have a better way of handling historical billing cycles (https://github.com/cloudquery/plugin-sdk/pull/569), so maybe we can update it to use that when it's ready.